### ISSUE TITLE
[feature request] Task tags

### ISSUE THREAD

**Issue by uqix** [ansible-semaphore/semaphore#1845]
![image](https://github.com/ansible-semaphore/semaphore/assets/10142314/4a33cbdf-222d-424a-b28a-318562edd6dd)

and REST API `POST /project/{project_id}/tasks` needs a new request field `tags` to specify them.

**Comment by fiftin** [comment]
Do you mean Ansible tags?

**Comment by uqix** [comment]
No, I mean semaphore tasks have their own tags (e.g. `frontend` `backend` `app1` `app2`) that are specified at creation time, so that we can easily filter them in task list page.
